README.md

```diff
@@ -97,8 +72,7 @@ The following decoded JSON web token
 **Payload**
 ```javascript
 {
-  "ver": "0.1.0",
-  "type": "embed",
```

Comment:
Yes. v1.0 APIs do not use type in the tokens anymore. Tokens are only used in Embed.